lib/committee/parameter_converter.rb

```diff
@@ -0,0 +1,44 @@
+module Committee
+  class ParameterConverter
```

Comment:
Would you mind if we renamed this to `ParameterCoercer` and the method `#convert!` to `#call`?